be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -359,7 +359,7 @@ class FileColumnIterator final : public ColumnIterator {
     std::unique_ptr<PageDecoder> _dict_decoder;
 
     // keep dict page handle to avoid released
```

Comment:
also update this comment

Reply:
> also update this comment

seems no problem? we do need to keep it for some time :)